apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -205,9 +205,13 @@ export class SendMessageEmail extends SendMessageBase {
       to: email,
       subject,
       html,
-      from: command.payload.$sender_email || integration?.credentials.from || 'no-reply@novu.co',
+      from:
```

Comment:
Dp we have an e2e test that shows this overrides functionality works? Would be great to have it